pyEpiabm/pyEpiabm/tests/test_unit/test_routine/test_simulation.py

```diff
@@ -58,8 +58,6 @@ def test_configure(self, mock_mkdir):
             self.assertIsInstance(test_sim.population, pe.Population)
             del test_sim.writer
         mo.assert_called_with(filename, 'w')
-        mock_mkdir.assert_called_once_with(os.path.join(os.getcwd(),
-                                           self.file_params["output_dir"]))
```

Comment:
Is this mock no longer being called then? I think if we still have the mock we should check it is being called in the right way?

Reply:
This is only called if the folder doesn't exist and needs to be created. Was called before because the writers deleted the output folder and recreated them, but I removed this otherwise multiple writers couldn't share the same output folder.